Apps/APN-UIKit/APN UIKit/APN UIKit/View/LoginViewController.swift

```diff
@@ -22,18 +23,31 @@ class LoginViewController: UIViewController {
         super.viewDidLoad()
         
         configureLoginRouter()
+        addUserInteractionToSettingsImageView()
     }
     
     func configureLoginRouter() {
         let router = LoginRouter()
         loginRouter = router
         router.loginViewController = self
     }
+    
+    func addUserInteractionToSettingsImageView() {
+        let gestureOnSettings: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(LoginViewController.settingsTapped))
+
+        settings.addGestureRecognizer(gestureOnSettings)
+        settings.isUserInteractionEnabled = true
```

Comment:
Note that this is code that could reference a UIImageView extension function. As mentioned in a previous PR comment.

Reply:
👍